data/dynos_mgr_pack.cpp

```
@@ -114,6 +117,16 @@ void DynOS_Pack_SetEnabled(PackData* aPack, bool aEnabled) {
             DynOS_Tex_Deactivate(_Tex);
         }
     }
+
+    // Check if active goddard head changed and reload if needed
+    SysPath _NewActiveHead = Goddard_CalculateActiveMarioHeadBin();
+    if (_NewActiveHead != goddard_get_active_mario_head_bin()) {
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)